packages/origin-device-registry-api/src/device-registry/device.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { IExternalDeviceId } from '@energyweb/origin-backend-core';
+import { ExtendedBaseEntity } from '@energyweb/origin-backend-utils';
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ name: 'device_registry_device' })
+export class Device extends ExtendedBaseEntity {
+    constructor(device: Partial<Device>) {
+        super();
+        Object.assign(this, device);
+    }
+
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column()
+    owner: string;
+
+    @Column({ unique: true })
+    externalRegistryId: string;
+
+    @Column({ unique: true })
+    smartMeterId: string;
+
+    @Column('simple-json', { nullable: true })
+    externalDeviceIds?: IExternalDeviceId[];
```

Comment:
Do we still need this when we have externalRegistryId?

Reply:
I'd rather keep it, for some other project we also used real-time data id or something.